src/main/java/duke/task/Task.java

```diff
@@ -30,6 +30,13 @@ public void markDone() {
         done = true;
     }
 
+    /**
+     * Marks the task as not done.
+     */
+    public void markUnDone() {
```

Comment:
Unnecessary public. Please change it

Reply:
Thanks for spotting it, I have fixed it along with other access rights for some variables